### PR TITLE
[14] : Create a stub for parser

### PR DESCRIPTION
**Pull request by Zhunusau**
Added project GodelMastery.FleaMarket.HtmlParser
Added project GodelMastery.FleaMarket.HtmlParserInterfaces
Added project GodelMastery.FleaMarket.Kufar
Added HtmlParserProvider in GodelMastery.FleaMarket.BL.Core.Helpers.HtmlParserHelper

### CODE REVIEW

File: GodelMastery.FleaMarket/GodelMastery.FleaMarket.BL/Dtos/LotDto.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GodelMastery.FleaMarket.BL.Dtos
+{
+    public class LotDto
+    {
+        public int Id { get; set; }
```
Comment (kpavaliayeu) [inline GodelMastery.FleaMarket/GodelMastery.FleaMarket.BL/Dtos/LotDto.cs]: Please, add ID to the external item (ID from Kufar)

File: GodelMastery.FleaMarket/GodelMastery.FleaMarket.BL/Core/Helpers/HtmlParserHelper/IHtmlParserProvider.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GodelMastery.FleaMarket.BL.Dtos;
+
+namespace GodelMastery.FleaMarket.BL.Core.Helpers.HtmlParserHelper
+{
+    interface IHtmlParserProvider
+    {
+        ICollection<LotDto> GetLotDtosByFilter(FilterDto filterDto);
```
Comment (kpavaliayeu) [inline GodelMastery.FleaMarket/GodelMastery.FleaMarket.BL/Core/Helpers/HtmlParserHelper/IHtmlParserProvider.cs]: Rename to GetLotsByFilter(...). Don't use 'Dto' in name.

File: GodelMastery.FleaMarket/GodelMastery.FleaMarket.Kufar/KufarHtmlLot.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using GodelMastery.FleaMarket.HtmlParserInterfaces;
+
+namespace GodelMastery.FleaMarket.Kufar
+{
+    public class KufarHtmlLot : IHtmlLot
+    {
+        public string Name { get; set; }
```
Comment (kpavaliayeu) [inline GodelMastery.FleaMarket/GodelMastery.FleaMarket.Kufar/KufarHtmlLot.cs]: Is that unique ID of the lot on the external source (Kufar)? If true - please, rename to Id

File: GodelMastery.FleaMarket/GodelMastery.FleaMarket.BL/Dtos/LotDto.cs (hunk 1 above)
Comment (Zhunusau) [reply GodelMastery.FleaMarket/GodelMastery.FleaMarket.BL/Dtos/LotDto.cs]: added property SourseId

File: GodelMastery.FleaMarket/GodelMastery.FleaMarket.Kufar/KufarHtmlLot.cs (hunk 3 above)
Comment (Zhunusau) [reply GodelMastery.FleaMarket/GodelMastery.FleaMarket.Kufar/KufarHtmlLot.cs]: added new property SourceId

**Pull request merged**